src/components/00-materials/icons-raw/key.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none"/><path d="M12.65 10C11.83 7.67 9.61 6 7 6c-3.31 0-6 2.69-6 6s2.69 6 6 6c2.61 0 4.83-1.67 5.65-4H17v4h4v-4h2v-4H12.65zM7 14c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2z" fill="currentColor"/></svg>
```

Comment:
fill="none" - do we need that or is it possible to remove?

Reply:
That's how they come from material design.
https://material.io/resources/icons/?style=baseline

Reply:
Ok. And you just picked the icons from there? No AXA own icons?

Reply:
Most Axa icons are from there. Is an illusion that axa makes is own icons ;)

Reply:
And yes it is needed. Otherwise the icons look like this. (for key and message i removed it)
<img width="240" alt="Screenshot 2020-03-11 at 08 39 08" src="https://user-images.githubusercontent.com/38284563/76393415-ca9e9200-6373-11ea-8650-aa297858f3dc.png">